CHANGELOG.md

```diff
@@ -22,6 +23,12 @@
 
 # Changelog
 
+## 2017.10.04.0
+
+* Added Apache2 license.
+* Repository moved to /mesosphere/dcos-e2e.
+* Add 2 new functions - run and popen, which have customisable users.
```

Comment:
* Added `run`, which is similar to `run_as_root` but takes a `user` argument.
* Added `popen`, which can be used for running commands asynchronously.